src/main/java/com/github/onsdigital/dp/authorisation/permissions/models/Policy.java

```diff
@@ -1,19 +1,21 @@
 package com.github.onsdigital.dp.authorisation.permissions.models;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
+/**
+ * Policy.
+ */
 public class Policy {
-    String ID;
-    public Condition condition;
+    private Condition condition;
+    private String ID;
 
     public Policy(String id) {
-        ID = id;
+        this.ID = id;
     }
-
     public Policy(String id, Condition condition) {
         ID = id;
         this.condition = condition;
     }
+    public Condition getCondition() {
+        return condition;
+    }
+
```

Comment:
The linter still complains that the value of the field Policy.ID is not used. To fix this you could add a getID method e.g. 
```suggestion
public String getID() {
      return this.ID;
  }
```